### ISSUE TITLE
User profile page

### ISSUE THREAD

**Issue by JAWD-001** [JAWD-001/peer_mentoring#30]
Recent posts aren't populating on user profile page

**Comment by bbelderbos** [comment]
@JAWD-001 I don't think we checked this out today, please debug using breakpoint and let me know

pybites-search results (truncated)

│ Using pdb to Inspect Python Objects                            │ https://www.youtube.com/watch?v=OWGyfBBwpSM                       │
│ How to Use Pdb to Debug Your Code                              │ https://pybit.es/pdb-debugger.html                                │

**Comment by bbelderbos** [comment]
@JAWD-001 is this still an issue?

**Comment by JAWD-001** [comment]
Yes

**Comment by JAWD-001** [comment]
I just tried and still couldn't figure it out

**Comment by bbelderbos** [comment]
What did you try so far? Any code commit?

**Comment by JAWD-001** [comment]
Hey, will look into this tonight after writing some more tests with pytest
